Update userguide

potatowagon opened the pull request:
change date time format
change edit fin commands to edit/fin tasktype index
remove daily schedule view

Commit:
change edit and del in features to match the current project's command format

Commit:
removed schedule view
partially edit time format to include :

Commit:
change date format in format date

pyokagan commented on docs/UserGuide.md:
```diff
@@ -99,15 +97,21 @@
 
 11. To delete a task, try:
+    	
+        del float 1
```
This will unfortunately require some code restructuring in the parser :-(. (Waiting on @nightism for the parser rework :-) )

Can we go with `del-float` and `edit-float` for now?

potatowagon replied:
sure, will change it, but since they wont be checking the userguide tmr, would u rather i do something for CRUD?

potatowagon replied:
it okay if theres nothing left to assign

pyokagan replied:
Have to submit userguide and devguide to IVLE today.

pyokagan replied:
I'm doing the UML model diagrams now for #40

potatowagon replied:
okay, will polish it up

pyokagan commented:
@potatowagon Can get this done by 10pm? I need to submit a copy to IVLE

Commit:
no message

Commit:
Quickstart: Edit language to sound engaging
Quickstart: change the examples so they are coherent, to make quickstart a walkthrough.
Quickstart: correct date and edit del format to eg. edit-float del-float

pyokagan commented:
@potatowagon Okay unfortunately I need to merge this PR right now so I can submit the user guide and developer guides to IVLE

potatowagon commented:
can i have 20 more minutes? sorry

pyokagan commented:
@potatowagon Okay, 20 more minutes starting from now, make it quick.

pyokagan commented on docs/UserGuide.md:
```diff
@@ -84,30 +81,23 @@
-    ![alttext](./images/userguide/index.png "")
-
-    Enter anything into command line to hide index view.
+    ![alttext](./images/userguide/index.png "")to update with actual app shots
```
@potatowagon `to update with actual app shots`?????

potatowagon replied:
yup, can be done with the current ui PR but i need time

potatowagon replied:
or else i can use the mock app

potatowagon replied:
still i need to redo all the images or it wont make sense, i can get it done by latest 1120

Commit:
Features: fix view all and view <date>
Features: fix edit-float del-float

Commit:
FEATURES: fix summary table formatiing and content

potatowagon commented:
ahh i still need to add in pictures

potatowagon commented:
wordings all fix, give me 20 sorry

pyokagan commented:
@potatowagon Nevermind, no point rushing. I'll just touch up what we have at 435e45e3deda12af7a2bf9034cfa9c51fab9401b and submit for now.

Continue working on this PR.

Commit:
add images for coherency
fix quickstart summary

pyokagan reviewed (approved):
Will merge after my nap today.

pyokagan commented:
Merged in 6970401e42862ccf02e08821c0482bed0d2b4116
